one_fm/one_fm/custom/issue.json

```diff
@@ -11,37 +11,37 @@
    "collapsible": 0,
    "collapsible_depends_on": null,
    "columns": 0,
-   "creation": "2022-02-20 09:58:54.462988",
+   "creation": "2022-03-17 16:30:52.624064",
    "default": null,
    "depends_on": null,
    "description": null,
    "docstatus": 0,
    "dt": "Issue",
    "fetch_from": null,
    "fetch_if_empty": 0,
-   "fieldname": "pivotal_tracker",
-   "fieldtype": "Data",
```

Comment:
Pivotal Tracker field seems missing from the custom field

Reply:
👍

Reply:
@pjamsheer @mymi14s Not sure what happened with this branch.
Raising a PR from a new branch for this.